src/docs/accounts/quotas/manage-event-stream-guide.mdx

```diff
@@ -41,7 +41,7 @@ For more information and code samples checkout:
 
 The integration attaches global handlers to capture uncaught exceptions (`onerror`) and unhandled rejections (`onunhandledrejection`). Both handlers are enabled by default but can be disabled through configuration. For more information see [GlobalHandlers Integration](/platforms/javascript/#globalhandlers)
 
-Checkout additional configuration options with the [TryCatch](/platforms/javascript/#trycatch) and [ReportingObserver](/platforms/javascript/#reportingobserver) integrations.
+Check out additional configuration options with the [TryCatch](/platforms/javascript/#trycatch) and [ReportingObserver](/platforms/javascript/#reportingobserver) integrations.
```

Comment:
oof. I see that we're missing the heading for TryCatch in /platforms/javascript/integrations/default/. I will fix this. Suggest we link to the page for now.
```suggestion
Check out additional configuration options with the [TryCatch](/platforms/javascript/integrations/default/) and [ReportingObserver](/platforms/javascript/integrations/plugin/#reportingobserver) integrations.
```